### ISSUE TITLE
Get face encoded from face recognizer to save to in the database

### ISSUE THREAD

**Issue by dangvu5998** [justadudewhohacks/face-recognition.js#54]
Face recognition can recognize face but it save data traning in a model but I want to get face descriptor vector from library to save it in the database. How can i do that?

**Comment by bobmoff** [comment]
you can serialize the model and get an array of objects. each object contains a property named facedescriptors. is that what you mean ?

**Comment by dangvu5998** [comment]
Yes. But i want somehow get facedescriptors directly for my custom.
